api-reference/docstubs/api/microsoft.fast.pi.viewgenerator.v2.profile.models-profile-get.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Get profile"
+description: "Read properties and relationships of a profile object."
+author: "**TODO: Provide Github Name**"
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# Get profile
+
+Namespace: Microsoft.Fast.PI.ViewGenerator.V2.Profile.Models
+
+Read properties and relationships of a [profile](../resources/microsoft.fast.pi.viewgenerator.v2.profile.models-profile.md) object.
```

Comment:
@wandeg , @dkershaw10 , sent the following to you in email, but in case you missed the email, wanted to take the chance to call out the profile resource as an example whose navigation properties are indeed returned in a GET.

AFAIK, internally they implemented it such that the navigation properties can auto-expand by including an [AutoExpand](https://docs.microsoft.com/en-us/odata/webapi/autoexpand-attribute) attribute.

Curious if we plan to differentiate between this type of nav properties, and those of most other entities in Graph which don't auto-expand?

Reply:
Not sure if that goes into annotations (it's not part of the main CSDL AFAIK).  Again would be manual :(